<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetrePrincipale</class>
 <widget class="QDialog" name="FenetrePrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>861</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tabConsulter">
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <attribute name="title">
     <string>Consulter stock</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>291</width>
       <height>129</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEditAjoutDesignationArticle">
        <property name="placeholderText">
         <string>Libelle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEditAjoutPoidsArticle">
        <property name="inputMask">
         <string>9999.99</string>
        </property>
        <property name="placeholderText">
         <string>Poids</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="boutonAjoutArticle">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ajout</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEditAjoutCodeArticle">
        <property name="placeholderText">
         <string>Article</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEditAjoutEmplacementArticle">
        <property name="placeholderText">
         <string>Emplacement</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSupprimer">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEditSupprimerArticle">
        <property name="placeholderText">
         <string>Produit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="boutonSupprimer">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabModifier">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEditModifierArticle">
        <property name="placeholderText">
         <string>Article</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="boutonModifier">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>251</width>
       <height>119</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="2" column="0">
       <widget class="QLabel" name="ModificationEmplacement_2">
        <property name="text">
         <string>Désignation</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditModificationDesignationArticle"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditModificationPoidsArticle"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ModificationCodeArticle">
        <property name="text">
         <string>Code article</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditModificationEmplacementArticle"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditModificationCodeArticle"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="ModificationEmplacement">
        <property name="text">
         <string>Emplacement</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ModificationEmplacement_3">
        <property name="text">
         <string>Poids</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonValidationModification">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabRechercher">
    <attribute name="title">
     <string>Rechercher</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>Article ou emplacement</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="boutonRechercher">
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Consulter fiche produit</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEditRechercher">
        <property name="placeholderText">
         <string>Article</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="boutonConsulterFicheProduit">
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>652</width>
       <height>150</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Désignation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Poids</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Emplacement</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QTextBrowser" name="afficheEmplacementArticle"/>
      </item>
      <item row="0" column="3">
       <widget class="QTextBrowser" name="afficheDesignationArticle"/>
      </item>
      <item row="1" column="1">
       <widget class="QTextBrowser" name="affichePoidsArticle"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="codeArticle">
        <property name="text">
         <string>Code article</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextBrowser" name="afficheCodeArticle"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Creation d'utilisateur</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>160</width>
       <height>103</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditeCreationLogin">
        <property name="placeholderText">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditeCreationMotDePasse">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxChoixDroitUtilisateur">
        <item>
         <property name="text">
          <string>Logisticien</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Administrateur</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCreationUtilisateur">
        <property name="text">
         <string>Validez</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
